app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -19,7 +19,7 @@
   }
 
   @include media-down(mobile) {
-    &.gem-c-breadcrumbs--collapse-on-mobile .gem-c-breadcrumbs--item {
+    &.gem-c-breadcrumbs--collapse-on-mobile .gem-c-breadcrumbs__item {
```

Comment:
changing this class will mean [all of these other repos](https://github.com/search?q=org%3Aalphagov+.gem-c-breadcrumbs--item&type=Code) will require a change aswell

Reply:
Eeek. Thanks, good spot.

Reply:
Actually looks like just collections. I'll raise a PR there to change it.

Reply:
@DilwoarH have raised a PR in collections (https://github.com/alphagov/collections/pull/796) - are you happy to approve this PR?